lib/liquid/strainer.rb

```diff
@@ -26,7 +26,7 @@ class << self
     end
 
     def self.add_filter(filter)
-      raise ArgumentError, "Expected module but got: #{f.class}" unless filter.is_a?(Module)
+      raise ArgumentError, "Expected module but got: #{filter.class}" unless filter.is_a?(Module)
```

Comment:
Yep, it's a bug I happened to come across. Decided to fix it along the way. The test is already there, `test/integration/strainer_test.rb` I'll just move it to unit tests, as discussed above.